flutter_screen_recording_web/lib/flutter_screen_recording_web.dart

```diff
@@ -72,27 +147,43 @@ class WebFlutterScreenRecording extends FlutterScreenRecordingPlatform {
       this.mediaRecorder = new MediaRecorder(stream, {'mimeType': mimeType});
 
       this.mediaRecorder.addEventListener('dataavailable', (Event event) {
-        print("datavailable ${event.runtimeType}");
         recordedChunks = JsObject.fromBrowserObject(event)['data'];
         this.mimeType = mimeType;
-        print("blob size: ${recordedChunks?.size ?? 'empty'}");
       });
 
+      _onStopCompleter = new Completer<String>();
+
+      displayMediaStream.addEventListener("inactive", (event) {
+        if (mediaRecorder != null && mediaRecorder.state != "inactive") {
```

Comment:
I think we should use condition as typeScript:
`mediaRecorder !== null && mediaRecorder.state !== "inactive"`

Reply:
Cannot do in a dart file

Reply:
Oh forget it, i was reviewing js file at that time :)))